.github/workflows/python-test-python-version.yml

```diff
@@ -0,0 +1,57 @@
+on:
+  workflow_call:
+
+defaults:
+  run:
+    working-directory: ./python
+
+jobs:
+  python_tests:
+    name: Python ${{ matrix.py_version }} Unit Tests
+    runs-on: ubuntu-22.04
+
+    strategy:
+      matrix:
+        py_version: ["3.9.13"] # py_version: ["3.9.13", "3.10.5", "3.11.7"]
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Set up Python ${{ matrix.py_version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.py_version }}
+
+      - name: Install the AaC Rest-Api Package
+        run: pip install -e .
+
+      - name: Install requirements
+        run: pip install -r requirements.txt
+
+      - name: Run Tox Tests
+        run: tox -e $(echo py${{ matrix.py_version }} | awk -F "." '{print $1$2;}')
+
+      - name: Upload Code Coverage Report
+        uses: actions/upload-artifact@v3
+        with:
+          name: test-coverage-report
+          path: python/html_code_coverage/
+
+      # - name: Run Tox Tests With Coverage
+      #   if: matrix.py_version == '3.11.7'
+      #   run: tox -e py311 -- --coverage-report html
+
+      # - name: Upload Code Coverage Report
+      #   uses: actions/upload-artifact@v3
+      #   if: matrix.py_version == '3.11.7'
+      #   with:
+      #     name: test-coverage-report
+      #     path: python/html_code_coverage/
+
+      - name: Upload logs from failed runs
+        if: ${{ failure() }}
+        uses: actions/upload-artifact@v3
+        with:
+          name: failed_test_log
+          path: python/src/aac/aac.log
```

Comment:
```suggestion
```
Don't believe we have access to the `aac.log` in the standalone plugin repos. It doesn't seem to get generated from what I recall.